.travis.sh

```diff
@@ -0,0 +1,10 @@
+#! /bin/bash
+
+set -e -x
+
+make -C control check
```

Comment:
Um, the same command is used in the [`%check` section in the spec file](https://github.com/yast/skelcd-control-openSUSE-LangAddOn/blob/master/package/skelcd-control-openSUSE-LangAddOn.spec#L52). I wanted to run the very same command here.

Moving to `rake check:syntax` sounds like a good idea, I'll look at that...